doc/changelogs/CHANGELOG_V14.md

```diff
@@ -51,7 +51,7 @@ including the autocompleted part. Pressing `<escape>` cancels that behavior.
 
 #### Support for Top-Level Await
 
-It's now possible to use the await keyword outside of async functions.
+It's now possible to use the await keyword outside of async functions, with the `--experimental-top-level-await` flag.
```

Comment:
It's now possible to use the await keyword outside of async functions, with the `--experimental-top-level-await` or. `--harmony-top-level-await` flag.

Reply:
we would not like to recommend that people use harmony flags